### DISCUSSION TITLE
This is the least advanced xray mod ever

### DISCUSSION THREAD

**Discussion by Somebody-Somewhere-In-USA** [AdvancedXRay/XRay-Mod#276] [General]
This is the dumbest xray mod I have ever seen. You have to add each and every block to the see-through list individually. You do know that most xray mods do the opposite, right? Usually every block will be invisible and you can toggle the ones you want to be visible. That is WAY less work for the player.

Also, toggling xray doesn't actually let you see anything because there is this annoying outline on all of the invisible blocks, and there are so many invisible blocks that the outlines of them all merge into a big black area. You can't see anything except black!

Also also, it only seems to affect a small number of chunks very close to the player. What good is seeing through only the blocks right next to you?

**Comment by MichaelHillcox** [comment]
Use a different one then, k bye
